### ISSUE TITLE
Bumping patch on merge to default branch

### ISSUE THREAD

**Issue by Physium** [PaulHatch/semantic-version#162]
I understand there are commit / branch patterns but I'm looking for a way to simply bump patch version whenever I merge to default (main).

**Comment by PaulHatch** [comment]
This action only looks at the Git repo itself, so you'd need to use the merge to drive something else like a tag or message. You can you the combination of `bump_each_commit` and `bump_each_commit_patch_pattern` to trigger increments to the patch pattern from a message, but you still need to make sure that commit message is there.

**Comment by Physium** [comment]
Fair enough, but how would it work if I use a feature branch merge-rebase strategy?

can i assume its something like this?
- push a commit with defined patch pattern
- merge to default branch
- action detects that among the recent bunch of commits that there is a commit message with defined pattern
- bumps patch
